extensions/2.0/Vendor/EXT_materials_bump/README.md

```diff
@@ -0,0 +1,142 @@
+# EXT\_materials\_bump
+
+## Contributors
+
+- Ben Houston, Threekit [@bhouston](https://github.com/bhouston)
+
+Copyright 2023 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft, Ratified by the Khronos Group
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Exclusions
+
+- This extension must not be used on a material that also uses `KHR_materials_pbrSpecularGlossiness`.
+- This extension must not be used on a material that also uses `KHR_materials_unlit`.
+
+## Overview
+
+This extension adds one parameters to the metallic-roughness material: `bump`.
+
+`bump` allows users to perturb the surface normal via a bump map. The bump map is a grayscale texture, where the value of each texel represents the height of the surface at that texel.  The height is then used to perturb the normal direction.
+
+Benefits of bump maps are primarily these two:
+
+1. Bump maps are much more compact that normal maps and lead to smaller glTF files as well as less GPU memory requirements.  This is because the normal map has to store a full 3D vector per texel, while the bump map only needs to store a single scalar value per texel.  Thus bump maps require 3 times less data than normal maps for transmission and GPU storage.  Normal maps are also very sensitive to texture compressions methods, while bump maps are less sensitive.
```

Comment:
For purposes of glTF users, reducing GPU memory would require use of GPU compression formats, correct? If I'm reading `KHR_texture_basisu` correctly, that implies that `bumpTexture` must use the red channel, like `occlusionTexture`

Reply:
If we do not pack the bump map into another channel of an existing common glTF texture map then yes, we would need to use a compressed texture if we want to reduce GPU memory requirements.

Reply:
Can I see the evidence that bump maps are less sensitive to compression artifacts than normals maps? I'm surprised because producing a normal map from a bump map is effectively taking a derivative, and derivatives amplify noise.

Reply:
> Can I see the evidence that bump maps are less sensitive to compression artifacts than normals maps?

Normal maps can store information that bump maps struggle to store, but for things that bump maps can store, they work well.  Normal maps are more general purpose and historically been a struggle to reconstruct the floating point normals with high accuracy from compressed textures that just store 8-bit representations.  https://developer.download.nvidia.com/whitepapers/2008/real-time-normal-map-dxt-compression.pdf

Bump maps do not claim to be able to do the beautiful slow moving large scale smooth curves and shapes that normal maps do.  Rather they are for fine detail.  Using normal maps for fine detail is overkill in terms of memory usage.

Reply:
Okay, for 8-bit lossless images I see your point. I was thinking of lossy compression mostly, but we can't usually make use of that for normal maps anyway.

Reply:
There are specific texture compression formats that are suitable for normal maps - such as those mentioned in the paper I linked in my previous comment.

But because bump map are used for micro details, generally you do not magnify them much at all.  At that size on screen you do not really seem compression artifacts.  Think of a noise texture that you would put on a piece of concrete or a fine ridged bump map you would put on commercial-grade carpet, or a bump map on shirt fabric - the details are so fine that you can not see any compression artifacts.

Whereas with normal maps they are usually representing larger scale details, large smooth surfaces, like the wrinkles on the shirt, and you do magnify them, thus the compression artifacts are much more visible.

The high frequency of the detail bump maps make it harder to see compression artifacts unlike the low frequency details in the normal map.

Reply:
I have changed the extension to use the red channel per @donmccurdy's suggestion.